### DISCUSSION TITLE
Keywallet Touch Wallet (Project ID: 7dc07a15346a0111be317c99354f7c75) Project Submission

### DISCUSSION THREAD

**Discussion by KwonWonHyun** [WalletConnect/walletconnect-monorepo#2892] [Developer Support]
Hello, I am a developer from Keywallet Touch.

We are currently in the process of migrating to Wallet Connect v2, and I just submitted a new entry to announce our support for v2 on the Explorer.

If you have some time, could you please review it?

The Project ID is <7dc07a15346a0111be317c99354f7c75>.

Thank you.

**Comment by JeroldG** [comment]
Hi therethe , I just reviewed your submission and unfortunately we were unable to test the connection via Walletconnect. Can you please check on your end? Link to test app that we use: https://lab.web3modal.com/
